### ISSUE TITLE
H1 and subhead style for form intro pages

### ISSUE THREAD

**Issue by bethpottsVADEPO** [department-of-veterans-affairs/va.gov-team#19780]
## Issue Description
A decision was made for accessibility reasons to add the form number to the intro page to distinguish it from the previous page. This was so there wouldn't be confusion about whether the user had actually moved to the next step in the process of applying. We're revisiting the subhead as a result, since the subhead immediately under the H1 has been referring to the form number as well. So there's repetition. 

**Sitewide content recommends this approach to address the issue:**

**H1 that refers to the type of benefit (VA Form number in parentheses)**
Subhead is the exact name of the form

**Apply for VA education benefits (VA Form 22-1990)**
Application for VA Education Benefits

**Apply for benefits under the National Call to Service program (VA Form 22-1990N)**
Application for VA Education Benefits Under the National Call to Service (NCS) Program

**Apply to use transferred education benefits (VA Form 22-1990E)**
Application for Family Member to Use Transferred Benefits

**Change your education benefits (VA Form 22-1995)**
Request for Change of Program or Place of Training

**Apply for dependent education benefits (VA Form 22-5490)**
Dependents’ Application for VA Education Benefits

**Change your dependent education benefits (VA Form 22-5495)**
Dependents’ Request for Change of Program or Place of Training

**Apply for the Rogers STEM Scholarship (VA Form 22-10203)**
Application for Edith Nourse Rogers STEM Scholarship

**Apply for VET TEC (VA Form 22-0994**)
Application for Veteran Employment Through Technology Education Courses (VET TEC)

**File for disability compensation (VA Form 21-526EZ)**
Application for Disability Compensation and Related Compensation Benefits

**File a Benefits Delivery at Discharge claim (VA Form 21-526EZ)**
Application for Disability Compensation and Related Compensation Benefits
Note: Benefits Delivery at Discharge (BDD)—I don’t think this is a separate form, so the above 526 form would cover this. Need to confirm.

**File for disability compensation (VA Form 21-526EZ)**
Application for Disability Compensation and Related Compensation Benefits

**Request a Higher-Level Review (VA Form 20-0996)**
Higher-Level Review

**Apply for Veteran Readiness and Employment (VA Form 28-1900)**
Application for Vocational Rehabilitation for Claimants with Service-Connected Disabilities

**Apply for Personalized Career Planning and Guidance (VA Form 28-8832)**
Education/Vocational Counseling Application (Chapter 36)

**Request help with VA debt payments (VA Form 5655)**
Financial Status Report

---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by jenniferlee-dsva** [comment]
Is there any way that the form number can be moved down to the subhead? A couple of concerns w/this approach: 

- We call for a standard of 50 chars on H1s on hub pages; and 70 on RS articles. So this approach would create a lot of exceptions. 
- Our H1 font is GIANT and the longer titles will def take up a ton of space, and likely wrap several lines. 
- In mobile, these long H1s will def wrap many many times, pushing content down. 

If we want to make application intro page H1s an exception and have no limit, I think that would be one approach, but I would still worry about scannability, wrapping, and pushing important content down. 

(And I understand if 508 issues make this impossible; just wanted to raise these concerns in case a workaround is possible.)

**Comment by DanielleThierryUSDSVA** [comment]
We hear you and have considered these same concerns, @jenniferlee-dsva! 

This is specifically for accessibility, yes. The form number in the subhead wouldn't solve the problem that led to this, which is that there is then no change in the H1 from the original intro page (with wizard) to the actual form page. 

I do think one thing we can add to the guidelines is to keep the H1 CTA as short as possible and/or within a specific character count. As part of this, we could revisit some of the above super-long ones above and make recommendations to try to shorten them. @bethpottsVADEPO, thoughts on that?

**Comment by DanielleThierryUSDSVA** [comment]
Examples:

Apply for VA education benefits (VA Form 22-1990N)
Application for VA Education Benefits Under the National Call to Service (NCS) Program

**Comment by bethpottsVADEPO** [comment]
I agree that we need to revisit some of these--and we can work to make existing H1s shorter. Danielle and I have been talking about what the standard should be for H1s. In tier 1, the H1 limit is 52. In R&S, it's 70. So we need to nail down what the limit should be for forms. We'll let you know where we land.

**Comment by RLHecht** [comment]
Update: Looks like Beth and Danielle need further discussion on what the H1 character limit will be for forms. Once that's done we would need to ask the product teams to implement these changes on the form intro pages.

**Comment by mistymg** [comment]
Ticket was moved to Closed column but not actually closed.
